src/Hazelcast.Net/Sql/HazelcastSqlException.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Hazelcast.Exceptions;
+
+namespace Hazelcast.Sql
+{
+    public class HazelcastSqlException : HazelcastException
+    {
+        public Guid ClientId { get; }
```

Comment:
I think we name it's named `ClientId` in other clients, but `MemberId` in C#? @zpqrtbnk, should I use `MemberId` everywhere?